.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @procore-oss/procore-ospo
```

Comment:
do we want to add the [procore-rspec-profiling](https://github.com/orgs/procore-oss/teams/procore-rspec-profiling) team as codeowners here?

Reply:
I hadn't created the team yet because who will be on it?  Do we know who the owners are yet?  I can create it and add myself and you for now?

Reply:
I take it back.  Adding.